src/wpt/harness.ts

```diff
@@ -285,12 +336,12 @@ function shouldRunTest(message) {
   return true;
 }
 
-function prepare(options) {
+function prepare(options: Options) {
```

Comment:
when eslint is run on this file, we will receive errors saying that each function return type should be explicitly defined.